lib/maze/server.rb

```diff
@@ -35,26 +28,23 @@ class << self
       # @return [String] The UUID attached to all command requests for this session
       attr_reader :command_uuid
 
+      def set_status_code_generator(generator, verb = DEFAULT_HTTP_VERB)
+        @generators ||= {}
+        @generators[verb] = generator
```

Comment:
It feels wrong to me to only set POST when no verb is given

I don't _think_ it could actually cause bugs, but I'd feel more comfortable if not passing a verb was equivalent to setting it for all verbs rather than just POST

Reply:
Or is the assumption in existing tests using the all verbs step that it actually only applies to POST and not e.g. OPTIONS?

Reply:
> if not passing a verb was equivalent to setting it for all verbs rather than just POST

Yes, I think you're right and the current implementation definitely doesn't work in some cases (I believe when OPTIONS get involved).  Will rework along the lines you suggest.